Dockerfile.template

```diff
@@ -71,8 +71,21 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 	&& \
 	# we can change from world to world-bin in newer releases
 	make world && \
-	make install-world
-
+	make install-world && \
+	# clones pg_cron extension for local use, this is not available wihout installing
+ 	git clone https://github.com/citusdata/pg_cron.git /pg_cron && \
+	cd pg_cron && make && make install && \
```

Comment:
additionally, can we also make sure the PATH is correct for users to use it? 

from their docs, the `pgsql-15` would need to work for postgres10+, so something like `pgsql-"$PG_MAJOR"`. i would need to check if that nomenclature has changed
```
git clone https://github.com/citusdata/pg_cron.git
cd pg_cron
# Ensure pg_config is in your path, e.g.
export PATH=/usr/pgsql-15/bin:$PATH
make && sudo PATH=$PATH make install
```